files/sdw-admin.py

```diff
@@ -70,7 +69,7 @@ def install_pvh_support():
     TODO: install this via package requirements instead if possible
     """
     try:
-        subprocess.run(["sudo", "qubes-dom0-update", "-y", "-q", "grub2-xen-pvh"])
+        subprocess.run(["sudo", "qubes-dom0-update", "-y", "-q", "grub2-xen-pvh"], check=False)
```

Comment:
We might want `check=True` or another subprocess method (`subprocess.check_call` ?) if the intention is to error if the grub2-zen-pvh installation doesn't work. (This isn't introduced by your PR, but ruff is making it noticeable.)

(Also, not really your problem since it was preexisting, but if you're in here anyway and open to random nits, I just noticed a typo in the error message on like 74: "Error installing grub2-xen-pvah" should be "Error installing grub2-xen-pvh", no "a".)

Reply:
Good catch. This was in my "Automatic ruff fixes" commit, so I think was automatically edited when I ran `ruff check --fix`. And sure enough, if you look at the docs for [subprocess.run()](https://docs.python.org/3/library/subprocess.html#subprocess.run), the default is `check=False`, which means it's always been false. I'll change to `subprocess.check_call()` like the other functions in this file use.